### PR TITLE
domination: init at 1.2.3

### PR DESCRIPTION
**Pull request by fgaz**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/games/domination/default.nix [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  };
+
+in stdenv.mkDerivation {
+  pname = "domination";
+  version = "1.2.3";
+
+  # The .zip releases do not contain the build.xml file
+  src = fetchsvn {
+    url = "https://svn.code.sf.net/p/domination/code/Domination";
+    rev = "1964";
```
Comment (SuperSandro2000) [inline pkgs/games/domination/default.nix:34]: Where can you find this number? How is this going to work with nixpkgs-update? What happens if it bumps version?

File: pkgs/games/domination/default.nix [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    jdk8
+    ant
+    makeWrapper
+  ];
+
+  buildPhase = "ant";
+
+  installPhase = ''
+    # Remove unnecessary files and launchers (they'd need to be wrapped anyway)
+    rm -r \
```
Comment (SuperSandro2000) [inline pkgs/games/domination/default.nix:48]:
```suggestion
    rm \
```

I don't think directories are removed.

Comment (fgaz) [reply pkgs/games/domination/default.nix:48]: .app is a directory

File: pkgs/games/domination/default.nix (hunk 1 above)
Comment (fgaz) [reply pkgs/games/domination/default.nix:34]: Right, I should add a comment. The repo doesn't seem to have tags, so the update is manual unfortunately. nixpkgs-updater will not work (it checks that the package actually changes iirc)

Comment (fgaz) [reply pkgs/games/domination/default.nix:37]: Added a comment

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 104499` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>domination</li>
  </ul>
</details>

**Comment by fgaz** [comment]
Ping

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 104499` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>domination</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 104499` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>domination</li>
</ul>
</details>

**Pull request merged**